library/aur.py

```diff
@@ -253,13 +256,15 @@ def main():
     else:
         use = params['use']
 
-    if params['upgrade'] and (params['name'] or use == 'makepkg'):
-        module.fail_json(msg="Upgrade cannot be used with this option.")
+    if params.get('upgrade', False) and use == 'makepkg':
+        module.fail_json(msg="Upgrade cannot be used with the helper 'makepkg'.")
```

Comment:
I would not call _makepkg_ a helper, helpers are not supported by Arch while _makepkg_ is supported

Reply:
Fair enough! What about changing that text to `tool`?

Reply:
ok